doc/api.md

```diff
@@ -742,6 +744,60 @@ Unreserves a path for a publishing application. Returns success or failure only.
 - If a path reservation exists for the supplied base_path but a different
   publishing_app, the command will fail.
 
+## `PUT /publish-intent(/*base_path)`
+
+[Request/Response detail][put-intent-pact]
+
+Used to create or update a publishing intent for a particular base_path and
+reserve the base path in the publishing-api. A publishing intent sets when the TTL
+cache should be cleared.
```

Comment:
The terminology isn't quite right here - the goal of the intent is to reduce caching of *content* at the specified time. It's actually a bug that the TTL itself is also cached by Fastly :-|.

Reply:
Well technically it's not reducing the cache on content as it's just reducing the cache for a certain base_path as the content isn't there yet, but i see your point and i will make it clearer! :)